shakedown/dcos/package.py

```diff
@@ -74,6 +74,12 @@ def install_package(
 
     cosmos = _get_cosmos()
     pkg = cosmos.get_package_version(package_name, package_version)
+    if app_id is None:
+        properties = pkg.config_json().get('properties')
+        if 'service' in properties:
+            app_id = properties['service']['properties']['name']['default']
```

Comment:
marathon:   https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/M/marathon/4/marathon.json.mustache#L3

marathon-lb: https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/M/marathon-lb/14/marathon.json.mustache#L2

these are 2 different ways of setting the "app_id".   They are NOT controlled by the user.

however:
This is the service name: https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/M/marathon/4/marathon.json.mustache#L42

by convention the ['labels']['DCOS_SERVICE_NAME'] is the same as app_id

This is pulling things out of the config.json which we have no convention for.